src/Steam.Common/SteamId.cs

```diff
@@ -0,0 +1,492 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace Steam
+{
+    /// <summary>
+    /// A globally unique identifier for Steam accounts, groups, lobbies, and chat rooms
+    /// </summary>
+    public readonly struct SteamId : IEquatable<SteamId>
+    {
+        private const uint AccountIdMask = 0xFFFFFFFF;
```

Comment:
We should consider whether we want to add documentation to private members as well.

Reply:
I don't think it's necessary, unless the private member is incredibly vague we may add a comment but I don't think we should add too many docs.